api/k8s/v1beta4/k8s_helpers.go

```diff
@@ -80,7 +80,7 @@ func (s *API) WaitForClusterPools(req *WaitForClusterPoolsRequest) error {
 	}
 
 	for _, pool := range pools.Pools {
-		err = s.WaitForPool(&WaitForPoolRequest{
+		_, err := s.WaitForPool(&WaitForPoolRequest{
```

Comment:
pool object should be used, and `WaitForClusterPools ` should return []*Poll

Reply:
Hmm cf https://github.com/scaleway/scaleway-sdk-go/pull/334 ?